contributing_to_docs/doc_guidelines.adoc

```diff
@@ -152,58 +156,39 @@ Use the relative file path (from the file you are editing, to the file you are l
 
 ----
-link:../../architecture/core_concepts/routes.adoc
+xref:../../architecture/core_concepts/routes.adoc#architecture-core-concepts-routes
 ----
 ====
 
 .Markup example of cross-referencing to internal topics
 ----
 Rollbacks revert part of an application back to a previous deployment. Rollbacks can be performed using the REST API or
-the link:../cli_reference/get_started_cli.html#installing-the-cli[OpenShift CLI].
+the xref:../cli_reference/get_started_cli.adoc#installing-the-cli[OpenShift CLI].
 
-Before you can create a domain, you must first link:../dev_guide/new_app.html[create an application].
+Before you can create a domain, you must first xref:../dev_guide/new_app.adoc#new-app[create an application].
 ----
 
 .Rendered output of cross-referencing to internal topics:
 ====
-Rollbacks revert part of an application back to a previous deployment. Rollbacks can be performed using the REST API or the link:../cli_reference/get_started_cli.html#installing-the-cli[OpenShift CLI].
-
-Before you can create a domain, you must first link:../dev_guide/new_app.html[create an application].
-====
-
-For comparison and clarification, the following example shows the previous method of linking to other topics.
-Do not use the old method of linking to internal topics, as shown in the following examples.
-
-.Example of old method of linking to other topics (not recommended)
-----
-Rollbacks revert an application back to a previous deployment and can be performed using the REST API or
-the OpenShift CLI. See the link:cli.html#deployment-rollbacks[CLI documentation] for more information.
-
-Before you can create a domain, you must first create an application. See link:applications.html#create_app[Create an Application]
-for information on how to create an application.
-----
-
-This example renders like so:
-====
-Rollbacks revert an application back to a previous deployment and can be performed using the REST API or the OpenShift CLI. See the link:cli.html#deployment-rollbacks[CLI documentation] for more information.
+Rollbacks revert part of an application back to a previous deployment. Rollbacks can be performed using the REST API or the xref:../cli_reference/get_started_cli.adoc#installing-the-cli[OpenShift CLI].
 
-Before you can create a domain, you must first create an application. See link:applications.html#create_app[Create an Application] for information on how to create an application.
+Before you can create a domain, you must first xref:../dev_guide/new_app.adoc#new-app[create an application].
 ====
 
-The difference here is that in the first example the link to [create an application] is part of the actual sentence. In the second example, both links are sentences that begin with "See....", which is not recommended.
-
 === Links to External Websites
 
 If you want to link to a different website, use:
 
 ----
-http://othersite.com/otherpath[friendly descriptor]
+link:http://othersite.com/otherpath[friendly descriptor]
```

Comment:
I'd `s/descriptor/reference text/` since "descriptor" sounds very technical.  I know this was part of the original text, but since we're here, might as well...